lib/docs/src/components/table.stories.tsx

```diff
@@ -320,6 +342,14 @@ export const RowClassName = (): JSX.Element => {
 	);
 };
 
+Utils.story(RowClassName, {
+	desc: `
+Moai table provides user with \`RowClassName\` prop. This prop receives a
```

Comment:
Like above, this is not the "user". We can just skip the target (developers) here.

A good way to start writing about a prop is to write about its functionality, thinking of the developer as the 2nd party. Something like "You can set a class name for a table's rows via the "rowClassName" prop". The second sentence should describe the function itself (like you did)